front/components/assistant/conversation/actions/AgentMessageActionsDrawer.tsx

```diff
@@ -9,40 +9,25 @@ import {
 
 import { getActionSpecification } from "@app/components/actions/types";
 import type { ActionProgressState } from "@app/lib/assistant/state/messageReducer";
-import { useConversationMessage } from "@app/lib/swr/conversations";
-import type {
-  AgentActionType,
-  LightAgentMessageType,
-  LightWorkspaceType,
-} from "@app/types";
+import type { AgentActionType, LightWorkspaceType } from "@app/types";
 
 interface AgentMessageActionsDrawerProps {
-  conversationId: string;
-  message: LightAgentMessageType;
+  messageActions: AgentActionType[];
   actionProgress: ActionProgressState;
   isOpened: boolean;
   isActing: boolean;
   onClose: () => void;
   owner: LightWorkspaceType;
 }
 export function AgentMessageActionsDrawer({
-  conversationId,
-  message,
+  messageActions,
   actionProgress,
   isOpened,
   isActing,
   onClose,
   owner,
 }: AgentMessageActionsDrawerProps) {
-  const { message: fullAgentMessage, isMessageLoading } =
-    useConversationMessage({
-      conversationId,
-      workspaceId: owner.sId,
-      messageId: isOpened ? message.sId : null,
-    });
-
-  const actions =
-    fullAgentMessage?.type === "agent_message" ? fullAgentMessage.actions : [];
+  const actions = messageActions;
```

Comment:
We should keep that hook call , for the case where  actions are coming from a conversations endpoint, which do not contains actions details. Actions details should only be embedded when the message is coming from an event.